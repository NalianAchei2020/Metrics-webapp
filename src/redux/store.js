import { configureStore } from '@reduxjs/toolkit';
import countrySlice from './country/countrySlice';

const store = configureStore({
  reducer: {
    countries: countrySlice,
  },
});

export default store;
