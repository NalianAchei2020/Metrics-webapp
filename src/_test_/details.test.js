import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Details from '../pages/details';

it('Mission should render', () => {
  const details = render(
    <Provider store={store}>
      <BrowserRouter>
        <Details />
      </BrowserRouter>
    </Provider>,
  );
  expect(details).toMatchSnapshot();
});
