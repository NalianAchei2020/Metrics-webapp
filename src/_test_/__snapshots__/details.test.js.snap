// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mission should render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="datail-container"
        >
          <div
            class="im-containter"
          >
            <h1
              class="headin-text"
            >
              none
            </h1>
            <img
              class="flags-img"
            />
          </div>
          <div
            class="status-bar"
          >
            <h2
              class="country-status"
            >
              City/Town BreakDown - 2023
            </h2>
          </div>
          <h2
            class="itemS-Info"
          >
            <span>
              Name:
            </span>
            none
          </h2>
          <h2
            class="itemS-Info"
          >
            <span>
              Capital:
            </span>
            none
          </h2>
          <h2
            class="itemS-Info"
          >
            <span>
              Continent:
            </span>
            none
          </h2>
          <h2
            class="itemS-Info"
          >
            <span>
              Population:
            </span>
            none
          </h2>
          <h2
            class="itemS-Info"
          >
            <span>
              Area:
            </span>
            none
            km²
          </h2>
          <h2
            class="itemS-Info"
          >
            <a
              href="none"
            >
              Map
            </a>
          </h2>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="datail-container"
      >
        <div
          class="im-containter"
        >
          <h1
            class="headin-text"
          >
            none
          </h1>
          <img
            class="flags-img"
          />
        </div>
        <div
          class="status-bar"
        >
          <h2
            class="country-status"
          >
            City/Town BreakDown - 2023
          </h2>
        </div>
        <h2
          class="itemS-Info"
        >
          <span>
            Name:
          </span>
          none
        </h2>
        <h2
          class="itemS-Info"
        >
          <span>
            Capital:
          </span>
          none
        </h2>
        <h2
          class="itemS-Info"
        >
          <span>
            Continent:
          </span>
          none
        </h2>
        <h2
          class="itemS-Info"
        >
          <span>
            Population:
          </span>
          none
        </h2>
        <h2
          class="itemS-Info"
        >
          <span>
            Area:
          </span>
          none
          km²
        </h2>
        <h2
          class="itemS-Info"
        >
          <a
            href="none"
          >
            Map
          </a>
        </h2>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
